content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -25,7 +25,7 @@ The .NET Tracer library for Datadog is open-source. For more information see the
 The .NET Tracer supports automatic instrumentation on .NET 5, .NET Core 3.1, and .NET Core 2.1. It also supports [.NET Framework][2].
 
 <div class="alert alert-warning"> 
-  <strong>Note:</strong>  If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.
+  <strong>Notes:</strong><br>- Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API only allows one subscriber (for example, APM). Datadog strongly recommends only running a single APM solution within your applications' environment to ensure maximum visibility.<br>- If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.
```

Comment:
Formatting nit: shouldn't this be a real unsorted list (that renders to `<ul><li>`) instead of faking it with `<br>-`?
```suggestion
  <strong>Notes:</strong>
- Datadog automatic instrumentation relies on the .NET CLR Profiling API. This API only allows one subscriber (for example, APM). Datadog strongly recommends only running a single APM solution within your applications' environment to ensure maximum visibility.
- If you are using both automatic and custom instrumentation, it is important to keep the package versions (for example, MSI and NuGet) in sync.
```

Reply:
Please ignore if you tried this already and it doesn't work inside a `<div>` 😅

Reply:
Moving our conversation from Slack to the PR. 

This will break the font layout of the alert box.